tests/unit/resource_manager_tests.cpp

```diff
@@ -77,3 +79,38 @@ TEST(ResourceManager, getAllocatorInvalidId)
   ASSERT_THROW(rm.getAllocator(umpire::invalid_allocator_id),
                umpire::util::Exception);
 }
+
+TEST(ResourceManager, aliases)
+{
+  auto& rm = umpire::ResourceManager::getInstance();
+
+  auto alloc = rm.getAllocator("HOST");
+  rm.addAlias("HOST_ALIAS", alloc);
+  EXPECT_THROW({
+      rm.addAlias("HOST_ALIAS", alloc);
+  }, umpire::util::Exception);
+
+  auto alloc_alias = rm.getAllocator("HOST_ALIAS");
+  EXPECT_EQ(alloc_alias.getId(), alloc.getId());
+  EXPECT_EQ(alloc_alias.getName(), alloc.getName());
+  EXPECT_NE(alloc_alias.getName(), "HOST_ALIAS");
+
+  EXPECT_THROW({
+    rm.removeAlias("BLAH", alloc);
+  }, umpire::util::Exception);
+  EXPECT_NO_THROW({
+    rm.removeAlias("HOST_ALIAS", alloc);
+  });
+
+  alloc = rm.getAllocator("HOST");
```

Comment:
Is this not needed? (see line 87)

Reply:
Oh, I see: "After calling, allocator can no longer be accessed by calling getAllocator(name)." - but what if it really doesn't :O